cmd/livepeer_cli/wizard_stats.go

```diff
@@ -228,26 +228,14 @@ func (w *wizard) delegatorStats() {
 	fmt.Println("|DELEGATOR STATS|")
 	fmt.Println("+---------------+")
 
-	pendingStake := ""
-	if d.PendingStake.Int64() == -1 {
```

Comment:
Hm might be ok to keep this behavior for now [1] - while `pendingStake()` and `pendingFees()` no longer revert, its still possible for calls to run out of gas since remote ETH nodes (i.e. Infura) will usually set a gas cap on RPC calls (for example geth has a `-rpc.gascap` flag) for DDOS protection. According to the [docs](https://infura.io/docs/ethereum/json-rpc/eth_call), Infura currently sets the gas cap to 2x the block gas limit so it is unlikely that `pendingStake()` and `pendingFees()` would run up against this cap on mainnet. That being said, I don't feel great about making assumptions about the gas cap imposed by a remote ETH node and its also easier to run up the gas usage for these calls on a network with faster rounds such as Rinkeby. 

[1] I'm not sure if an out of gas error results in the same empty output error (which `GetDelegator()` currently checks for) that a revert error triggers

Reply:
If the node operates correctly then it should never run into the gascap, should it? 

So if it does we could assume someone has altered the livepeer node software to manifest such behaviour, no? Then you could argue that if you want to misuse our software that you better handle your own errors as well 😅 . 

Sorry for my possible ignorance here, I hadn't heard of infura imposing that on eth_calls

Reply:
> If the node operates correctly then it should never run into the gascap, should it?

The node might run into the gas cap is if an account has not claimed earnings in many rounds so `pendingStake()`/`pendingFees()` iterates through many rounds. As mentioned in my previous comment, this is unlikely to happen on mainnet, but just wanted to highlight it as a possibility.

Since this error handling is already here, I don't see any harm in just leaving it as is for now.

Reply:
Gotcha.

Reply:
Readded the previous behaviour but added unit formatting instead of just string conversion

```
	pendingStake := ""
	if d.PendingStake.Int64() == -1 {
		pendingStake = "Please fetch pending stake separately"
	} else {
		pendingStake = eth.FormatUnits(d.PendingStake, "LPT")
	}

	pendingFees := ""
	if d.PendingFees.Int64() == -1 {
		pendingFees = "Please fetch pending fees separately"
	} else {
		pendingFees = eth.FormatUnits(d.PendingFees, "ETH")
	}
```